### DISCUSSION TITLE
Can't select reviewers and labels in Pull&Request

### DISCUSSION THREAD

**Discussion by youngjun0627** [lablup/contribution-academy-2021#47] [Q&A] [answered]
### Describe the problem
[코스2](https://github.com/lablup/contribution-academy-2021/wiki/Challengers:2%EC%A3%BC%EC%B0%A8)의 8번까지 진행하였고, 9번 진행하는 과정에서 reviewers와 labels를 지정할 수가 없습니다. 혹시 몰라서 PR하고 draft상태에서도 시도를 해봤으나 설정 이모티콘이 보이지 않아 reviewers와 labels를 고칠수가 없어 현재 잠시 closed해놓은 상태입니다.

### To Reproduce
[코스2](https://github.com/lablup/contribution-academy-2021/wiki/Challengers:2%EC%A3%BC%EC%B0%A8)의 8번까지 진행하고, 9번 진행하는 과정에서 우측에 옵션이 뜨지 않습니다.

### Expected behavior
reviewers와 labels를 선택할 수 있을거라 기대했습니다.

### Screenshots
1. create PR 창에서
![image](https://user-images.githubusercontent.com/57058726/129145155-9efd9784-de56-4151-ab9a-24c0ffe749c5.png)
2. 존재하는 PR창에서
![image](https://user-images.githubusercontent.com/57058726/129145189-36b62883-845f-424f-ac33-0a6d9f16bc46.png)

### Environment Description

Browser version: Chrome - 92.0.4515.131(version)

**Comment by agatha197** [comment, marked as answer]
repository의 collaborator가 아닌 계정은 reviewer 지정과 tag 추가가 불가능한데, 잘못 안내해드린 것 같습니다 ㅠㅠ. 
Reviewers, Labels는 비워두시고 그대로 PR 보내주시면 됩니다 :) WIKI는 업데이트 했으며, 혼란을 드려 죄송합니다.
